### DISCUSSION TITLE
unicode mapping file is marked as unsupported

### DISCUSSION THREAD

**Discussion by joshi-mohit** [corazawaf/coraza#1022] [Q&A]
I see this as unsupported directive. Is there is any way to set the unicode codepoint -- "secunicodemap":            directiveUnsupported,

I am getting FP in 
Detects concatenated basic SQL injection and SQLLFI attempts, Matched Data: शिवा ,शिवा update 19 found within ARGS:json.value: शिवा ,शिवा update 19/3/24,शिवा update ,शिवा,पुन्हा कर्तव्य \xe0\xa4

The payload look like
{"key": "recent_search", "value": "\u0936\u093f\u0935\u093e ,\u0936\u093f\u0935\u093e update 19/3/24,\u0936\u093f\u0935\u093e update ,\u0936\u093f\u0935\u093e"}

This is possibly due to Devanagri script. Some of payloads in the website has ascii charcters.
How can we set the correct unicode mapping to have this fixed

Also see this FP on rule 942100--> SQL Injection Attack Detected via libinjection, Matched Data: 1c found within REQUEST_COOKIES:

**Comment by joshi-mohit** [comment]
@anuraaga 
@jptosso -- is there a way to have unicode directives updated in any way. This seems to be related to that or can you provide guidance on how to move on this

**Reply by jptosso** [reply]
It is, Unicode mapping is only applied at one transformation (Unicode) and the code is there, but commented.
I'm not sure if this is a space we wanna touch but this is easy to implement
